src/common/constants.ts

```diff
@@ -28,7 +27,63 @@ export const PYTH_USDC_USD_PRICE_ID_BETA = '0x41f3625971ca2ed2263e78573fe5ce23e1
 export const PYTH_USDC_USD_PRICE_ID_STABLE = '0xeaa020c61cc479712813461ce153894a96a6c00b21ed0cfc2798d1f9a9e9c94a';
 
 // Constants for Pimlico relayer
-export const FACTORY_ADDRESS_SIMPLE_ACCOUNT = '0x91E60e0613810449d098b0b5Ec8b51A0FE8c8985'
+export const FACTORY_ADDRESS_SIMPLE_ACCOUNT = '0x91E60e0613810449d098b0b5Ec8b51A0FE8c8985';
 
 // Temporary constants
-export const SUBGRAPH_HELPER_ADDRESS = "0xf012d32505df6853187170F00C7b789A8ecC41c2"
\ No newline at end of file
+export const SUBGRAPH_HELPER_ADDRESS = '0xf012d32505df6853187170F00C7b789A8ecC41c2';
+
+export const ARB_SEPOLIA_MARKET_COLLATERAL_PYTH_ID = new Map<string, string>([
```

Comment:
Simply rename this to `SYMBOL_TO_PYTH_FEED` as the Pyth price ids are same across all the networks and we can add mappings similar to `collateralMappingWithRegularSymbol` for getting the price ids for collaterals and markets.